tutorials/docs/101_optiml.md

```diff
@@ -0,0 +1,28 @@
+# WhizzML: 101 - Using an OptiML
+
+Following the schema described in the prediction workflow
+document, this is the code snippet that shows the minimal workflow to
+create an OptiML.
+
+```
+    ;; step 0: creating a source from the data in your remote "https://static.bigml.com/csv/iris.csv" file
+    (define source-id (create-source {"remote" "https://static.bigml.com/csv/iris.csv"}))
+    (log-info "Creating remote source: " source-id)
+    ;; step 1: creating a dataset from the previously created `source`
+    (define dataset-id (create-dataset source-id))
+    (log-info "Creating dataset from source: " dataset-id)
+    ;; step 2: creating an optiml. You can configure the maximum training
+    ;; time or other arguments to control the type of models used or set
+    ;; weight to be used. The example sets a `max_training_time" of 300 seconds
+    ;; and uses only decision trees and logistic regressions
+    (define optiml-id (create-optiml dataset-id
+                                     {"max_training_time" 300
+                                      "model_types" ["model"
+                                      "logisticregression"]}))
+    (log-info "Creating optiml from dataset: " optiml-id)
```

Comment:
sometimes we repeat that paragraph about not having to wait for a resource to finish, and sometimes like here, we don't mention that we don't need to wait for dataset-id.  i would perhaps remove the long explanation everywhere and add a comment inlined in the code, to avoid the repetition (although i see the idea is to make each doc self-contained, but for that maybe the comment `dataset-id ;; no need to explictly wait` could suffice?)

Reply:
well, it refers to dataset-id but also to source-id when creating a dataset. That's why I've preferred to add a general explanation instead of inline comments next to each resource create function.

Reply:
but in this document you are not including any explanation about not having to wait in create calls... why is this one different?

Reply:
oh, we could add that too. No problem.